vc_zoom/indico_vc_zoom/plugin.py

```diff
@@ -606,11 +600,24 @@ def _event_metadata_postprocess(self, sender, event, data, user=None, skip_acces
                 (visibility == 'registered' and user is not None and event.is_user_registered(user)) or
                 event.can_manage(user)
             ):
-                urls.append(format_link(assoc.vc_room.name, assoc.vc_room.data['url']))
+                items.append((assoc.vc_room.name, assoc.vc_room.data['url']))
             elif visibility == 'no_one':
                 # XXX: Not sure if showing this is useful, but on the event page we show the join link
                 # with no passcode as well, so let's keep the logic identical here.
-                urls.append(format_link(assoc.vc_room.name, assoc.vc_room.data['public_url']))
+                items.append((assoc.vc_room.name, assoc.vc_room.data['public_url']))
+
+        if not items:
+            return
+
+        if html:
+            desc = render_plugin_template('vc_zoom:event_metadata_desc.html', items=items)
+            return {'description': data['description'] + f'\n{desc}'}
+
+        if len(items) == 1:
+            urls = [url for name, url in items]
+            desc = f'Zoom: {urls[0]}'
+        else:
+            urls = [f'{name}: {url}' for name, url in items]
```

Comment:
```suggestion
            urls = [f' - {name}: {url}' for name, url in items]
```

To make it list-like?